spyder/widgets/mixins.py

```diff
@@ -263,6 +271,7 @@ def _format_text(self, title=None, signature=None, text=None,
         return template
 
     def _format_signature(self, signatures, parameter=None,
+                          width=_DEFAULT_WIDTH,

```

Comment:
Maybe  `max_width` is a better name as it is more clear what it implies (same as `max_lines`), it could be less than that but never larger. What do you think @dalthviz ?